### ISSUE TITLE
when validation, the loss is abnormal!

### ISSUE THREAD

**Issue by zj19921221** [sayakpaul/EvoNorms-in-TensorFlow-2#2]
hi  I am more familiar with tf_1.0 ; i chaned the code as i understand; but when i training the code ,the training phase is ok, but the validation is abnormal ,the loss is so so soso large; i guess the point of the moving_var;
as below is my code; can you please help me point out where is wrong in my code ?

```
def EvoNorm2dB0(x,` is_train=True, scope='EvoNorm2dB0', nonlinear=True, momentum=0.99, eps=1e-3):

        N, H, W, C = x.get_shape().as_list()
        with tf.variable_scope(scope, reuse=tf.AUTO_REUSE):
            gamma = tf.get_variable(name="gamma", shape=(1, 1, 1, C), initializer=tf.ones_initializer())
            beta  = tf.get_variable(name="beta",  shape=(1, 1, 1, C), initializer=tf.zeros_initializer())
            moving_var = tf.get_variable(name='moving_var', shape=(1, 1, 1, C), initializer=tf.zeros_initializer(), trainable=False)
        
            if nonlinear:
                v = tf.get_variable(name="v", shape=(1, 1, 1, C),initializer=tf.ones_initializer())
               
        
            if is_train:
                _, var = tf.nn.moments(x, [0, 1, 2], keep_dims=True)
                moving_averages.assign_moving_average(variable=moving_var, value=var, decay=momentum)
            else:
                var = tf.get_variable(name='moving_var', shape=(1, 1, 1, C), initializer=tf.zeros_initializer(), trainable=False)
        
            if nonlinear:
		den = tf.math.maximum(tf.sqrt(var + eps), v * x + instance_std(x))
		return x / den * gamma + beta
            else:
		return x * gamma + beta

```

**Comment by sayakpaul** [comment]
Hi. Is it possible to share a Colab Notebook to reproduce your issue?
